composer.json

```diff
@@ -29,12 +29,12 @@
         }
     ],
     "require": {
-        "php": "^8.0|^8.1",
-        "spatie/laravel-analytics": "^4.1"
+        "php": "^8.1",
```

Comment:
We also want to allow 8.2, right?

Reply:
`^8.1` will include `8.2`, and `8.3` for that matter - [composer caret](https://getcomposer.org/doc/articles/versions.md#caret-version-range-)